scripts/import_default_vpc.py

```diff
@@ -0,0 +1,81 @@
+from more_itertools import (
+    one,
+)
+
+from azul import (
+    config,
+    logging,
+)
+from azul.deployment import (
+    aws,
+)
+from azul.logging import (
+    configure_script_logging,
+)
+from azul.terraform import (
+    terraform,
+)
+
+log = logging.getLogger(__name__)
+
+
+def main() -> None:
+    if config.terraform_component == 'shared':
+        import_default_vpc()
+    else:
+        log.info("The 'shared' component is not selected, skipping import.")
+
+
+def get_default_vpc_id() -> str:
+    filters = [
+        {
+            'Name': 'isDefault',
+            'Values': ['true']
+        }
+    ]
+    vpcs = aws.ec2.describe_vpcs(Filters=filters)
+    return one(vpcs['Vpcs'])['VpcId']
+
+
+def get_default_security_group_id(vpc_id: str) -> str:
+    filters = [
+        {
+            'Name': 'vpc-id',
+            'Values': [vpc_id]
+        },
+        {
+            'Name': 'group-name',
+            'Values': ['default']
+        }
+    ]
+    groups = aws.ec2.describe_security_groups(Filters=filters)
+    return one(groups['SecurityGroups'])['GroupId']
+
+
+def import_default_vpc() -> None:
+    resources = terraform.run('state', 'list').splitlines()
+    found_default_vpc = 'aws_default_vpc.default' in resources
+    found_security_group = 'aws_default_security_group.default' in resources
+    vpc_id = (
+        None
+        if found_default_vpc and found_security_group else
+        get_default_vpc_id()
+    )
```

Comment:
```suggestion
    found_default_vpc = 'aws_default_vpc.default' in resources
    found_security_group = 'aws_default_security_group.default' in resources
    vpc_id = (
        None
        if found_default_vpc and found_security_group else
        get_default_vpc_id()
    )
```

This is all a little complicated. We typically use `@cached_property` to achieve this at a much lower cost.